pkg/sql/colexec/insert/insert.go

```diff
@@ -71,8 +72,8 @@ func handleWrite(n *Argument, proc *process.Process, ctx context.Context, bat *b
 		b.Clean(proc.Mp())
 	}
 	err := n.TargetTable.Write(ctx, bat)
+	atomic.AddUint64(&n.Affected, uint64(bat.Vecs[0].Length()))
```

Comment:
Should  increase this value even in case of error?

Reply:
done